.gitallowed

```diff
@@ -0,0 +1 @@
+xxxyyyzzz
```

Comment:
Can you give context here?

Reply:
Why do we need this checked in?

Reply:
Generally would say this is more a workstation config than a repository specific one similar to editor configs, which I usually agree shouldn't be checked in.
Any reason it should be checked in besides making live easier?
(Should that be enough of a reason. Actually not sure on our stance here as this relates to easier maintainer life.)